doc/conf.py

```diff
@@ -15,6 +15,9 @@
 from __future__ import print_function
 import sys
 import os
+
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
I don't understand. You build the docs of the development system using the old version?
You should make sure that the version you are building the docs from is the one that gets imported.